### ISSUE TITLE
I tried it on win7, but it's not worked!😔

### ISSUE THREAD

**Issue by HeyManMan** [icyguider/Shhhloader#15]
I tried it on win7 with cs4.4 payload. But it's not worked....

**Comment by icyguider** [comment]
Yeah so I've been aware of this issue for a while now and I have been able to replicate it. However, I have not had the time to debug and figure out a fix for it. While supporting Windows 7 would be nice, I'm not sure if it's a big concern for me at the moment when I have so many other things to work on. I hope you understand.

Closing this issue as it's essentially a duplicate of [this one](https://github.com/icyguider/Shhhloader/issues/7).
